charts/mirrorbits/templates/secret.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "mirrorbits.fullname" . }}
+type: Opaque
+data:
+  mirrorbits.conf: {{ .Values.mirrorbits.conf | b64enc }}
```

Comment:
why does this need to be a secret and not a configmap?

Reply:
because you have redis password in the configuration